features/display_help_request.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Show help request for admin
+  As an admin
+  In order to see tenants help request messages
```

Comment:
Not a blocker but this US does not make sense IMO `...In order to respond to messages from tenants.`